### ISSUE TITLE
More verbosity with specific toggles

### ISSUE THREAD

**Issue by mxmehl** [astroidmail/astroid#393]
There is at least one switchable toggle which the user isn't informed about if she activates it. There should be a message displayed if such a toggle is activated.

For example, `M-p` toggles auto-poll. But if this is pressed, there is no information about what has changes – only in the log:

`poll: toggle auto poll: false`

There should be a visible message popping up, for example in the bottom bar which is also used for the actions with marked threads/messages.

**Comment by gauteh** [comment]
Agreed, think we could use gtkinfobar or similar notification as nautilus
is using. Also for change clipboard target.

søn. 3. sep. 2017 kl. 23.04 skrev Max <notifications@github.com>:
